commons/src/main/java/com/powsybl/commons/config/PlatformConfig.java

```diff
@@ -117,7 +124,13 @@ public static ModuleConfigRepository loadDefaultModuleRepository() {
     }
 
     public static synchronized PlatformConfig defaultConfig() {
+        ServiceLoader<DefaultConfiguration> defaultConfigProvider = ServiceLoader.load(DefaultConfiguration.class);
+        if (Iterables.size(defaultConfigProvider) > 0) {
+            LOGGER.info("A default configuration is generated from an implementation.");
+            return Iterables.get(defaultConfigProvider, 0).get();
```

Comment:
Maybe we should log the implementation used in case of several implementation are found. Not really deterministic.

Consider changing the log level to debug